infra/postdeployment.tf

```diff
@@ -81,6 +82,6 @@ gcloud beta run jobs execute ${google_cloud_run_v2_job.client.name} --wait --pro
 echo "Warm up API"
 curl ${local.server_url}/api/products/?warmup
 
-# shutdown -h now
+shutdown -h now
```

Comment:
future: I realize it's a bit out of scope for the PR, but looking at this startup script, it took me a double-take to realize we probably don't want to run with `set -e` because we want all these things to happen, they don't depend on each other. It might be worth adding a comment to explain.

Reply:
I'm going to separate this and the other comment on this line into a Feature Request for later processing. Since metadata startup script runs async to TF, it can't block TF, so we'd need to have a separate process for that.

Reply:
https://github.com/GoogleCloudPlatform/terraform-dynamic-python-webapp/issues/50